website_sale_product_attachment/README.rst

```diff
@@ -0,0 +1,137 @@
+=============================
+eCommerce product attachments
+=============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fe--commerce-lightgray.png?logo=github
+    :target: https://github.com/OCA/e-commerce/tree/12.0/website_sale_product_attachment
+    :alt: OCA/e-commerce
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_product_attachment
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/113/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module lets you publish downloadable attachments in a product page.
+
+This is useful if you want to publish firmwares, manuals, specs, warranties,
+or whatever document related to the product.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+Instead of configuring attachments per product, to configure products per attachment:
+
+#. You need permissions to access system settings.
+#. Go to *Settings > Dashboard > Activate the developer mode*.
+#. Go to *Settings > Technical > Database Structure > Attachments*.
+#. Pick or create one.
+#. Use the field *Attached in product templates* to choose the products.
+
```

Comment:
Functional review 👍 

Just a minor change. I would add a little explanation that user should check "Is public document".

Reply:
Is that really needed? I don't think so.

Reply:
I tried both cases. If the checkbox is set, user can see the attachments. If the chechbox is not set, attachments are not appearing.

Did I miss something?

Reply:
Oh yes, good point.

Reply:
We have talked internally and the constraint is needed for avoiding leaks, so we are going to document it, but keep it.